src/app/components/media/MediaOrigin.js

```diff
@@ -118,10 +118,10 @@ const getIconAndMessage = (origin, user) => {
 const MediaOrigin = ({
   projectMedia: {
     media_cluster_origin: origin,
-    media_cluster_origin_user: { name: user } = {},
-  },
+    media_cluster_origin_user: user = {},
+  } = {},
```

Comment:
[comment] (non-blocker) I'm not a fan of this nested destructuring, but it works. Just a personal preference, I guess... :shrug: